src/quo2/components/calendar/calendar_month/view.cljs

```diff
@@ -0,0 +1,31 @@
+(ns quo2.components.calendar.calendar-month.view
+  (:require [react-native.core :as rn]
+            [quo2.theme :as theme]
+            [quo2.components.calendar.calendar-month.style :as style]
+            [quo2.components.buttons.button.view :as button]
+            [utils.number :as number-utils]
+            [quo2.components.calendar.calendar-month.utils :as utils]))
+
+(defn- calendar-month-internal
+  [{:keys [year month on-change]}]
+  (let [year  (number-utils/parse-int year)
+        month (number-utils/parse-int month)]
+    [rn/view
+     {:style style/container}
+     [button/button
+      {:icon                true
+       :type                :outline
+       :accessibility-label :previous-month-button
+       :size                24
+       :on-press            #(on-change (utils/previous-month year month))}
+      :i/chevron-left]
+     [rn/text {:style (style/text)} (utils/format-month-year year month)]
+     [button/button
+      {:icon                true
+       :accessibility-label :next-month-button
+       :size                24
+       :type                :outline
+       :on-press            #(on-change (utils/next-month year month))}
+      :i/chevron-right]]))
+
+(def calendar-month (theme/with-theme calendar-month-internal))
```

Comment:
The point of using this wrapper is so that you have access to 'theme' prop in this component- you can then pass that prop to the styles because anywhere theme is used we want to have this approach in place. For example for correct use of colors/theme-colors method you should be passing theme as the 3rd param -
E.g (colors/theme-colors <light-variant> <dark-variant> theme)